.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,21 +1,27 @@
-### Purpose of PR:
-Please give a short description of what this PR is for.
+### Purpose of PR  
+<!-- Describe what this PR does. -->
```

Comment:
Uncomment. 

I would like to see multiple PR templates, but we can make that a follow on issue